driver/src/main/java/org/kaazing/k3po/driver/internal/Robot.java

```diff
@@ -284,23 +287,40 @@ public void exceptionCaught(ChannelHandlerContext ctx, ExceptionEvent e) throws
         return new CompositeChannelFuture<>(channel, bindFutures);
     }
 
-    private void startConfiguration() {
+    private void startConfiguration() throws Exception {
```

Comment:
I believe if this exception is thrown and it is the only connection, then the startedFuture will never be set success or failure.  Meaning best case the test times out, if the developer remembered to put a timeout on the test.  Worse case it hangs.  Perhaps catch the exception in the call to startConfiguration or handle it differently